templates/frontend/objects/article_details.tpl

```diff
@@ -37,330 +29,300 @@
-			{foreach from=$pubIdPlugins item=pubIdPlugin}
-				{if $pubIdPlugin->getPubIdType() == 'doi'}
-					{continue}
+				{* DOI *}
+				{foreach from=$pubIdPlugins item=pubIdPlugin}
+					{if $pubIdPlugin->getPubIdType() != 'doi'}
+						{continue}
+					{/if}
+					{assign var=pubId value=$article->getStoredPubId($pubIdPlugin->getPubIdType())}
+					{if $pubId}
+						{assign var="doiUrl" value=$pubIdPlugin->getResolvingURL($currentJournal->getId(), $pubId)|escape}
+						<a href="{$doiUrl}">{$doiUrl}</a>
+					{/if}
+				{/foreach}
+
+				{* PubIds (other than DOI; requires plugins) *}
+				{foreach from=$pubIdPlugins item=pubIdPlugin}
+					{if $pubIdPlugin->getPubIdType() == 'doi'}
+						{continue}
+					{/if}
+					{assign var=pubId value=$article->getStoredPubId($pubIdPlugin->getPubIdType())}
+					{if $pubId}
+						<section>
+							<h2>
+								{$pubIdPlugin->getPubIdDisplayType()|escape}
+							</h2>
+							<div>
+								{if $pubIdPlugin->getResolvingURL($currentJournal->getId(), $pubId)|escape}
+									<a id="pub-id::{$pubIdPlugin->getPubIdType()|escape}" href="{$pubIdPlugin->getResolvingURL($currentJournal->getId(), $pubId)|escape}">
+										{$pubIdPlugin->getResolvingURL($currentJournal->getId(), $pubId)|escape}
+									</a>
+								{else}
+									{$pubId|escape}
+								{/if}
+							</div>
+						</section>
+					{/if}
+				{/foreach}
+
+				{* Article galleys *}
+				{if $primaryGalleys}
+					<section>
+						{foreach from=$primaryGalleys item=galley}
+							{include file="frontend/objects/galley_link.tpl" parent=$article galley=$galley purchaseFee=$currentJournal->getSetting('purchaseArticleFee') purchaseCurrency=$currentJournal->getSetting('currency')}
+						{/foreach}
+					</section>
 				{/if}
-				{assign var=pubId value=$article->getStoredPubId($pubIdPlugin->getPubIdType())}
-				{if $pubId}
-					<div class="item pubid">
-						<div class="label">
-							{$pubIdPlugin->getPubIdDisplayType()|escape}
-						</div>
-						<div class="value">
-							{if $pubIdPlugin->getResolvingURL($currentJournal->getId(), $pubId)|escape}
-								<a id="pub-id::{$pubIdPlugin->getPubIdType()|escape}" href="{$pubIdPlugin->getResolvingURL($currentJournal->getId(), $pubId)|escape}">
-									{$pubIdPlugin->getResolvingURL($currentJournal->getId(), $pubId)|escape}
-								</a>
-							{else}
-								{$pubId|escape}
+
+				{* Supplementary galleys *}
+				{if $supplementaryGalleys}
+					<section>
+						<h2 class="article-details-heading">{translate key="plugins.themes.healthSciences.article.supplementaryFiles"}</h2>
+						{foreach from=$supplementaryGalleys item=galley}
+							{include file="frontend/objects/galley_link.tpl" parent=$article galley=$galley isSupplementary="1"}
+						{/foreach}
+					</section>
+				{/if}
+
+				{* References *}
+				{if $parsedCitations->getCount() || $article->getCitations()}
+					<section>
+						<h2>
+							{translate key="submission.citations"}
+						</h2>
+						<ul>
```

Comment:
Don't you think it would be better to put references into the ordered list? I don't have a strong opinion here because we don't have article's full-text on the landing page.

Reply:
Fixed in commit https://github.com/pkp/pragma/commit/9a0be91cc6fda7b351550f2501d98fd5945cd7cc